### PR TITLE
py-virtualfish: added missed dependencies

### PR DESCRIPTION
**Pull request by bashu**
#### Description

Added missed dependencies: psutil

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.3 21G419 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-virtualfish/Portfile [hunk 1]
```diff
@@ -28,6 +28,7 @@ python.pep517_backend   poetry
 
 if {${name} ne ${subport}} {
     depends_lib-append  port:py${python.version}-virtualenv \
+                        port:py${python.version}-psutil \
```
Comment (reneeotten) [inline python/py-virtualfish/Portfile:31]: there are more dependencies listed than those in upstream's pyproject.toml file. Can you please verify and if there are indeed needed add them here explicitly?

**Comment by reneeotten** [comment]
ping @bashu

**Comment by bashu** [comment]
fixed @reneeotten

File: python/py-virtualfish/Portfile (hunk 1 above)
Comment (bashu) [reply python/py-virtualfish/Portfile:33]: fixed

**Pull request merged**